<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgIPChecker</class>
 <widget class="QWidget" name="dlgIPChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IP Checker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/IPChecker.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/IPChecker.qrc">:/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTItle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>64</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP Checker</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>-6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="lblCurrentIP">
       <property name="text">
        <string>Current &amp;IP:</string>
       </property>
       <property name="buddy">
        <cstring>txtCurrentIP</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCurrentIP">
       <property name="styleSheet">
        <string notr="true">background:yellow; color:black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="lblCompName">
       <property name="text">
        <string>Comp &amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>txtCompName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCompName"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblInterval">
       <property name="text">
        <string>&amp;Interval Period(mins):</string>
       </property>
       <property name="buddy">
        <cstring>spinInterval</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinInterval">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>&amp;Start/Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSettings">
       <property name="text">
        <string>Se&amp;ttings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnViewLog">
       <property name="text">
        <string>View &amp;Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblHelpMsg">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;To quit, right click the icon in system tray and select &quot;Quit&quot;.&lt;/span&gt;&amp;nbsp;&lt;a href&gt;IP Box Color codes...&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>&amp;Minimize to System Tray</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/IPChecker.qrc"/>
 </resources>
 <connections/>
</ui>
